src/main/scala/latis/service/hapi/HapiService.scala

```diff
@@ -21,10 +21,10 @@ import latis.catalog.Catalog
 import latis.server.ServiceInterface
 
 /**
- * A grouping of all four required HAPI endpoints.
+ * A grouping of all five required HAPI endpoints.
  *
- * The HAPI spec defines four required endpoints. This class groups
- * those four endpoints and a landing page and exposes them as a
+ * The HAPI spec defines five required endpoints. This class groups
+ * those five endpoints into a landing page and exposes them as a
```

Comment:
I wouldn't say this groups the endpoints *into* a landing page, because a landing page isn't composed of endpoints. The landing page and the endpoints are separate things, and this service groups them together into a single service.

Reply:
Oh I see that now. Sorry, I thought that `and` was a typo the first time I read it. I hope you don't mind if I add the word "together", that makes it read more clearly to me.